.github/workflows/delete-deployment.yml

```diff
@@ -49,6 +49,8 @@ jobs:
             echo "$app | last uploaded: $last_upload_date"
 
             if [[ -z $last_upload_date || $(date -d $last_upload_date +%Y-%m-%d) < $cut_off_date ]]; then
-              cf delete $app -rf
+              if [[ $app == prototype-preview* ]]; then
```

Comment:
That's really cool, I didn't know you could do a partial match like that with bash.
[I sniffed around a bit](https://ploegert.gitbook.io/til/os/unix/search-query/partial-string-matching-in-bash-scripts) to see what's going on.

Many of the examples I saw used quotes around the string with the wildcard outside.
Does this work?

And does that make
`$app == prototype-preview*`
and
`$app == "prototype-preview"*`
direct equivilents?

Reply:
Looks like they are equivalent:

```bash
~  $ docker run -it ubuntu bash
root@09698d9dd4d6:/# 
> for app in prototype-preview-deploy-protoype
> do
>   echo "$app"
>   if [[ $app == prototype-preview* ]]; then
>     echo deleted
>   fi
> done
prototype-preview-deploy-protoype
deleted
root@09698d9dd4d6:/#
> for app in prototype-preview-deploy-protoype
> do
>   echo "$app"
>   if [[ $app == "prototype-preview"* ]]; then
>     echo deleted
>   fi
> done
prototype-preview-deploy-protoype
deleted
root@09698d9dd4d6:/#
```

Reply:
I'm going to test opening and closing a PR with #127 to check that it still deletes - but that's a different path in this workflow anyway. Unfortunately there's no great way of testing the actions on something like a staging environment

Reply:
Sure, no prob - if it fails then can do follow up PRs until we get it right, there's no great harm here. 
Thanks for confirming!